lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -275,6 +278,22 @@ def extract_student(student, features):
                 (team.name for team in student.teams.all() if team.course_id == course_key),
                 UNAVAILABLE
             )
+
+        if include_enrollment_mode or include_verification_status:
+            print "getting enrollment/verification. student {0}, course key {1}".format(
+                student.id,
+                course_key
+            )
+            enrollment_mode = CourseEnrollment.enrollment_mode_for_user(student, course_key)[0]
+            if include_verification_status:
+                student_dict['verification_status'] = SoftwareSecurePhotoVerification.verification_status_for_user(
+                    student,
+                    course_key,
+                    enrollment_mode
+                )
+            if include_enrollment_mode:
+                student_dict['enrollment_mode'] = enrollment_mode
```

Comment:
Did this code already pre-exist elsewhere?  If so, rather than copying it here, it should be refactored so it can be reused.